<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KundenProfil</class>
 <widget class="QDialog" name="KundenProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Id:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditId">
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditName">
               <property name="maximumSize">
                <size>
                 <width>165</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>78</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelTravelId">
             <property name="text">
              <string>Travel Id:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTravelId">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>98</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>158</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButtonId">
             <property name="text">
              <string>Id</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonPrice">
             <property name="text">
              <string>Price</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonFromDate">
             <property name="text">
              <string>FromDate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtontoDate">
             <property name="text">
              <string>ToDate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSort">
             <property name="text">
              <string>Sort</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetTravel">
         <property name="minimumSize">
          <size>
           <width>411</width>
           <height>414</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>411</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetBooking">
         <property name="minimumSize">
          <size>
           <width>421</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>421</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="TabWidget">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="Flug">
            <attribute name="title">
             <string>Flug</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Id:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxFlugId">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Preis:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxFlugPreis">
                 <property name="maximum">
                  <double>9999999.990000000223517</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Vom:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDateEdit" name="dateEditFlugFrom">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Von:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEditFlugFrom"/>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Nach:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLineEdit" name="lineEditFlugTo"/>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Airline:</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QLineEdit" name="lineEditFlugAirline"/>
               </item>
               <item row="3" column="1">
                <widget class="QDateEdit" name="dateEditFlugTo">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>Bis:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEditFlugFromStadtError">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 0, 0);</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QLineEdit" name="lineEditFlugToStadtError">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 0, 0);</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="lineEditFlugFromStadt">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <widget class="QLineEdit" name="lineEditFlugToStadt">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Auto">
            <attribute name="title">
             <string>Auto</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Id:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxAutoId">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Preis:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxAutoPreis">
                 <property name="maximum">
                  <double>1000000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Vom:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDateEdit" name="dateEditAutoFrom">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Bis:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDateEdit" name="dateEditAutoTo">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Annahme</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEditAutoPickup"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Abgabe:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEditAutoReturn"/>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Company:</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="lineEditAutoCompany"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Hotel">
            <attribute name="title">
             <string>Hotel</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Id:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxHotelId">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Preis:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBoxHotelPreis">
                 <property name="maximum">
                  <double>999999.989999999990687</double>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Vom:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDateEdit" name="dateEditHotelFrom">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Bis:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QDateEdit" name="dateEditHotelTo">
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Hotel:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEditHotelName"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Stadt:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEditHotelTown"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>168</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSpeichern">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
